main/docs/get-started/applications/configure-jar.mdx

```diff
@@ -0,0 +1,157 @@
+---
+'og:description': Learn how to configure JWT-secured Authorization Requests (JAR)
+  for an application.
+'og:image': https://cdn2.auth0.com/docs/1.14553.0/img/share-image.png
+'og:title': Configure JWT-secured Authorization Requests (JAR)
+'og:url': https://auth0.com/docs/
+permalink: configure-jar
+title: Configure JWT-secured Authorization Requests (JAR)
+'twitter:description': Learn how to configure JWT-secured Authorization Requests (JAR)
+  for an application.
+'twitter:title': Configure JWT-secured Authorization Requests (JAR)
+---
+<Callout icon="file-lines" color="#0EA5E9" iconType="regular">

```

Comment:
Can we use CSS class names (theme tokens) instead of hard-coded colors for better maintainability?